client/src/Components/NavBarProfile/index.js

```diff
@@ -0,0 +1,141 @@
+import React, { Fragment } from 'react';
+import myStyles from './style';
+import { Grid, IconButton, Typography, Avatar } from '@material-ui/core';
+import NotificationsActiveIcon from '@material-ui/icons/NotificationsActive';
+import ArrowDropDownIcon from '@material-ui/icons/ArrowDropDown';
+import classNames from 'classnames';
+import TemporaryDrawerProfile from '../TemporaryDrawerProfile';
+import { Link } from 'react-scroll';
+import { Link as RouterLink } from 'react-router-dom';
+
+const NavBarProfile = () => {
+	const classes = myStyles();
+	return (
+		<Fragment>
+			<Grid
+				container
+				direction="row"
+				alignItems="center"
+				className={classes.grid}
+			>
+				<Grid item lg={1}></Grid>
+				<Grid
+					item
+					container
+					xs={12}
+					md={2}
+					alignContent="flex-start"
+					className={classes.imageGrid}
+				>
+					<img
+						className={classes.image}
+						src={require('./gsg_logo.png')}
+						alt="logopic"
+					/>
+				</Grid>
+				<TemporaryDrawerProfile />
+				<Grid
+					item
+					xs={12}
+					md={9}
+					wrap
+					alignContent="flex-end"
+					className={classes.menuGrid}
+				>
+					<Typography className={classes.typography}>
+						<Link
+							activeClass="active"
+							to="home"
+							spy={true}
+							smooth={true}
+							offset={0}
+							duration={500}
+							className={classes.link}
+						>
+							Home
+						</Link>
+						<Link
+							activeClass="active"
+							to="about"
+							spy={true}
+							smooth={true}
+							offset={0}
+							duration={500}
+							className={classes.link}
+						>
+							About
+						</Link>
+
+						<Link
+							activeClass="active"
+							to="steps"
+							spy={true}
+							smooth={true}
+							offset={0}
+							duration={500}
+							className={classes.link}
+						>
+							Steps
+						</Link>
+
+						<Link
+							activeClass="active"
+							to="mentors"
+							spy={true}
+							smooth={true}
+							offset={0}
+							duration={500}
+							className={classes.link}
+						>
+							Mentors
+						</Link>
+
+						<RouterLink to="/contact-us" className={classes.link}>
```

Comment:
why not to have it as a section in the home screen

Reply:
checkout the figma design and see where is the contact page would be